specs/phase0/fork-choice.md

```diff
@@ -101,6 +102,7 @@ class Store(object):
     finalized_checkpoint: Checkpoint
     best_justified_checkpoint: Checkpoint
     proposer_boost_root: Root
+    has_equivocated: Set[ValidatorIndex]
```

Comment:
`has_equivocated` reads like a predicate rather than a set. I suggest we name it `equivocated_indices` or `equivocating_indices`